packages/simple63/simple63.1/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+synopsis: "Integer compression and decompression module"
+description:
+  "Simple63 implements a module for compression and decompression of
+   non-negative integer sequences. It is similar to the Simple-8b
+   technique, adapted to work with OCaml's 63-bit integers."
+maintainer: ["mika@illouz.net"]
+authors: ["Mika Illouz"]
+license: "BSD"
+homepage: "https://github.com/barko/simple63"
+doc: "https://barko.github.io/simple63/"
+bug-reports: "https://github.com/barko/simple63/issues"
+depends: [
+  "dune" {> "2.6"}
+  "ocaml" {>= "4.08.0"}
+  "ppx_inline_test" {with-test}
+]
+build: [
+  ["dune" "subst"] {pinned}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@doc" {with-doc}
```

Comment:
```suggestion
    "@install"
    "@doc" {with-doc}
```
The previous question from @mseri still remains though. If no test is ran, `ppx_inline_test` is not required. We prefer to run the tests when possible if they are not unstable. Could you add `"@runtest" {with-test}` between those two lines to launch the tests if that's the case?

Reply:
Ok! Trying again and thanks for your patience.